src/main/java/presenters/helpFrame.java

```diff
@@ -0,0 +1,127 @@
+package presenters;
+
+import javax.swing.*;
+import java.awt.*;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.List;
+
+public class helpFrame extends JFrame implements ActionListener {
```

Comment:
Add a class docstring please

Reply:
Can you make the name searchpageFrame or sth so we dont mix it up with other jframes

Reply:
No problem, I will do that.